ets-demo/setup.py

```diff
@@ -281,8 +282,21 @@ def get_long_description():
         long_description=get_long_description(),
         long_description_content_type="text/x-rst",
         packages=setuptools.find_packages(include=["etsdemo", "etsdemo.*"]),
+        package_data={
+            "etsdemo": [
+                "images/*.png",
```

Comment:
i'm not sure if there is a manifest file in `ets-demo` but that we will probably want to update that as well.

Reply:
By adding the image in the `package_data`, this will include the file in the source distribution as well, so we might not need a manifest file yet?